v0/destinations/blueshift/transform.js

```diff
@@ -116,58 +113,29 @@ const identifyResponseBuilder = async (message, category, { Config }) => {
 };
 
 const groupResponseBuilder = async (message, category, { Config }) => {
-  if (!Config.usersApiKey) {
+  if (!Config.eventApiKey) {
     throw new CustomError(
-      "[BLUESHIFT] User API Key required for Authentication.",
+      "[BLUESHIFT] event API Key required for Authentication.",
       400
     );
   }
 
-  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  let payload = constructPayload(message, MAPPING_CONFIG[category.name]);
```

Comment:
```suggestion
  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
```

Reply:
You cannot reassign `const` but you can modify property of a `const` variable
Example 
```
const payload = { key: value };
payload.key2 = value
```
The above is allowed

```
const payload = { key: value };
payload  = { key2: value2 }
```
The above is not allowed


```
let payload = { key: value };
payload  = { key2: value2 }
```
The above is allowed